spec/models/post_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe Post, type: :model do
+  it "has a valid factory" do
+    expect(FactoryBot.build(:post)).to be_valid
+  end
+
+  it "is valid with a title and body" do
+    post = FactoryBot.build(:post)
+    expect(post).to be_valid
+  end
+
+  it "is invalid without a title" do
+    post = FactoryBot.build(:post,:invalid_title)
```

Comment:
Was using traits the recommended style for invalid data. Personally, I wouldn't create an invalid factory.
From memory something like `post = FactoryBot.build(:post, title: nil)` would also work and only be isolated to the test